docker-compose.yml

```diff
@@ -0,0 +1,52 @@
+version: "3.7"
+services:
+  load_balancer:
+    image: nginx:alpine
+    volumes:
+      - ~/permanent/devenv/nginx.conf:/etc/nginx/nginx.conf:ro
+      - ~/permanent/devenv/certs:/etc/ssl:ro
+    ports:
+      - "443:443"
+    depends_on:
+      stela:
+        condition: service_healthy
+  stela:
+    restart: always
+    build:
+      context: ../stela
+      dockerfile: Dockerfile.dev
+    ports:
+      - 8080:8080
+    env_file: 
+      - ../stela/.env
+    volumes:
+      - ../stela:/usr/local/apps/stela/dev
+      - ignore:/usr/local/apps/stela/dev/node_modules
+    depends_on:
+      database:
+        condition: service_healthy
+    healthcheck:
+      test: curl http://localhost:8080/api/v2/health
+  database:
+    image: postgres:14
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: permanent
+    healthcheck:
+      test: pg_isready
+    ports:
+      - "5432:5432"
+  database_setup:
+    image: postgres:14
+    volumes:
+      - ~/permanent/stela/database/base.sql:/base.sql:ro
```

Comment:
Let's use a relative path here as you do for other references to `stela`.

Reply:
I thought I saw a problem with using relative paths when working on this initially, but apparently that was a misdiagnosis, because it's working fine with relative paths now.